test/model/rotbaum/test_model.py

```diff
@@ -17,53 +17,28 @@
 
 # First-party imports
 from gluonts.model.rotbaum import TreePredictor
-
+from gluonts.evaluation.backtest import make_evaluation_predictions
+from gluonts.evaluation import Evaluator
+from gluonts.evaluation.backtest import backtest_metrics
 
 def test_accuracy(accuracy_test, dsinfo):
-    def quantile_loss(true, pred, quantile):
-        denom = sum(np.abs(true))
-        num = sum(
-            [
-                (1 - quantile) * abs(y_hat - y)
-                if y_hat > y
-                else quantile * abs(y_hat - y)
-                for y_hat, y in zip(pred, true)
-            ]
-        )
-        if denom != 0:
-            return 2 * num / denom
-        else:
-            return None
 
-    record = {}
     predictor = TreePredictor(
-        context_length=2, prediction_length=dsinfo["prediction_length"]
+        context_length=2, prediction_length=dsinfo["prediction_length"], freq = dsinfo["freq"]
     )
-    predictor_instance = predictor(dsinfo.train_ds)
-    data_for_pred = [
-        {
-            "start": ts["start"],
-            "target": ts["target"][: -dsinfo["prediction_length"]],
-        }
-        for ts in dsinfo.test_ds
-    ]
-    data_true = [
-        {
-            "start": ts["start"],
-            "target": ts["target"][-dsinfo["prediction_length"] :],
-        }
-        for ts in dsinfo.test_ds
-    ]
-    for quantile in [0.1, 0.5, 0.9]:
-        predictions = list(predictor_instance.predict(data_for_pred))
-        preds = [t.quantile(quantile) for t in predictions]
-        record[f"quantile_loss_{quantile}"] = quantile_loss(
-            list(chain(*[ts["target"] for ts in data_true])),
-            list(chain(*preds)),
-            quantile,
+    predictor(dsinfo.train_ds)
```

Comment:
We can add a `TODO` for a `train` method so it matches the `predictor = estimator.train` library structure.